### PR TITLE
Update sql-insights-overview.md

### PR DESCRIPTION
**Pull request by ujpat**

### CODE REVIEW

File: azure-sql/database/sql-insights-overview.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ To open SQL Insights (preview):
 
 For more instructions, see [Enable SQL Insights (preview)](sql-insights-enable.md) and [Troubleshoot SQL Insights (preview)](sql-insights-troubleshoot.md).
 
+**Note:** SQL insights are not supported for SQL Server On Prem Instances.
```
Comment (Court72) [inline azure-sql/database/sql-insights-overview.md:80]:
```suggestion
> [!NOTE]
> SQL Insights (preview) isn't supported for SQL Server on-premises instances.
```

**Comment by Court72** [comment]
@WilliamDAssafMSFT

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by WilliamDAssafMSFT** [comment]
@ujpat We already include a line in the opening table indicating that "Non-Azure instances** are not supported. This note doesn't appear to be necessary. Any reason we need this? Instead of a note, I'd rather add to the table that already lists supported/unsupported platforms.

**Comment by ujpat** [comment]
Thanks. Masha helped me to make the changes. We can close this out.

**Pull request closed without merging**